packages/ai-jsx/src/batteries/logging-integrations.tsx

```diff
@@ -0,0 +1,126 @@
+/**
```

Comment:
Thoughts on the module name?
Alternatively we can add this to `core/logs` as well.

Reply:
I'm fine with this for now.